chromium_src/ui/views/controls/focus_ring.cc

```diff
@@ -52,11 +46,17 @@ FocusRingTheme& GetFocusRingTheme() {
 
 }  // namespace
 
-#define setStyle                                                           \
-  setColor(color_.value_or(                                                \
-      GetFocusRingTheme().GetSystemColor(ColorIdForValidity(!invalid_)))); \
-  paint.setStyle
-
 #include "../../../../../ui/views/controls/focus_ring.cc"
 
-#undef setStyle
+namespace views {
+
+SkColor FocusRing::GetColor(View* focus_ring, bool valid) {
+  // To avoid unused GetColor() in anonymous ns error.
+  if (true) {
```

Comment:
Is this `if (true)` intentional?

Reply:
Yes, it's intentional to call below always.
W/o this, compiler will complain `::views::GetColor(focus_ring, valid)` is not used.